core/src/main/java/org/apache/calcite/sql/SqlDialect.java

```diff
@@ -504,8 +505,7 @@ public boolean supportsAggregateFunction(SqlKind kind) {
     return false;
   }
 
-  public boolean supportsFunction(SqlOperator operator, RelDataType type,
-                                  List<RelDataType> paramsList) {
+  public boolean supportsFunction(RexCall call) {
```

Comment:
Is the change to RexCall absolutely necessary? I don't want SqlDialect to depend on Rex or RelNode.